bitnami/redis/README.md

```diff
@@ -261,6 +261,7 @@ The following table lists the configurable parameters of the Redis<sup>TM</sup>
 | `sentinel.cleanDelaySeconds`                          | Delay seconds before issuing the the cleaning in the next node                                                                                      | `5`                                                     |
 | `sentinel.configmap`                                  | Additional Redis<sup>TM</sup> configuration for the sentinel nodes (this value is evaluated as a template)                                          | `nil`                                                   |
 | `sentinel.staticID`                                   | Enable static IDs for sentinel replicas (If disabled IDs will be randomly generated on startup)                                                     | `false`                                                 |
+| `sentinel.service.createMaster`                       | Create master service pointing to the current master                                                                                                | `false`                                                 |
```

Comment:
Just to maintain the order, I would place this row after the `sentinel.service.loadBalancerIP` to reflect the place where the parameter is appearing in the values.yaml